index.html

```diff
@@ -1,45 +1,203 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap"
+        rel="stylesheet" />
 </head>
+
 <body>
     <header>
         <a href="/" class="logo-text font-style">My Logo</a>
-        <img class="menu-icon" src="/assets/images/Enabled.png" alt="blue menu icone">
+        <img class="menu-icon" src="/assets/images/Enabled.png" alt="blue menu icone" />
     </header>
     <main class="container">
-       <div class="main">
-           <h1 class="main-heading font-style">I'm Nii Glad to see you!</h1>
-           <p class="main-description font-style">I’m a software developer! I can help you build a product , 
-               feature or website Look through some of my work and experience! 
-               If you like what you see and have a project you need coded, 
-               don’t hestiate to contact me.
+        <div class="main">
+            <h1 class="main-heading font-style">I'm Nii Glad to see you!</h1>
+            <p class="main-description font-style">
+                I’m a software developer! I can help you build a product , feature or
+                website Look through some of my work and experience! If you like what
+                you see and have a project you need coded, don’t hestiate to contact
+                me.
             </p>
             <h4 class="main-connect color font-style">Let's connect</h4>
             <div class="main-socials">
                 <a class="socials" href="https://twitter.com/_niicode">
-                    <img src="/assets/images/Vector-5.png" alt="symbol for twitter account">
+                    <img src="/assets/images/Vector-5.png" alt="symbol for twitter account" />
                 </a>
                 <a class="socials" href="https://linkedin.com/nii-darku-dodoo-082018148/">
-                    <img src="/assets/images/Vector-3.png" alt="symbol for linkedin account i">
-                    <img src="/assets/images/Vector-2.png" alt="symbol for linkedin account n">
+                    <img src="/assets/images/Vector-3.png" alt="symbol for linkedin account i" />
+                    <img src="/assets/images/Vector-2.png" alt="symbol for linkedin account n" />
                 </a>
                 <a class="socials" href="">
-                    <img src="/assets/images/Vector-1.png" alt="symbol for medium account">
+                    <img src="/assets/images/Vector-1.png" alt="symbol for medium account" />
                 </a>
                 <a class="socials" href="https://github.com/niicode">
-                    <img src="/assets/images/Vector-4.png" alt="symbol for github account">
+                    <img src="/assets/images/Vector-4.png" alt="symbol for github account" />
                 </a>
             </div>
-       </div>
+        </div>
+        <div class="main-projects">
```

Comment:
- [ ] it would be better if you use the `<section>` tag instead of the `<div>` tag that will give a better sense and also will improve your SEO